src/Bookings.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from "react";
+import Search from "./Search.jsx";
+import SearchResults from "./SearchResults.jsx";
+import NewBooking from "./NewBooking";
+
+const Bookings = () => {
+  const [bookings, setBookings] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState(null);
+  const [searchInput, setSearchInput] = useState("");
+  const [id, setID] = useState();
+
+
+  const filteredBookings = bookings.filter(
+    (person) =>
+      person.firstName.includes(searchInput) ||
+      person.surname.includes(searchInput)
+  );
+
+  function addBooking(newBooking) {
+    const newData = bookings.concat(newBooking);
+    setBookings(newData);
+  }
+
+  // https://raw.githubusercontent.com/CodeYourFuture/cyf-hotel-react/master/src/data/fakeBookings.json - previous api link
+  //https://cyf-react.glitch.me/error - error link
+  useEffect(() => {
+    fetch(`https://cyf-react.glitch.me/delayed`)
+      .then((response) => {
+        if (!response.ok) {
+          throw new Error("HTTP error, status = " + response.status);
+        }
+        return response.json();
+      })
+      .then((data) => {
+        setBookings(data);
+        setIsLoading(true);
```

Comment:
Does this work how it should? I think we should be setting loading state to false after our api call has finished 🤔